cypress/e2e/inventory/derive-fields-crud.cy.js

```diff
@@ -0,0 +1,128 @@
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../support/utils/stringTools';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import { JOB_STATUS_NAMES } from '../../support/constants';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import InventoryViewSource from '../../support/fragments/inventory/inventoryViewSource';
+
+describe('MARC -> MARC Bibliographic -> Derive MARC bib', () => {
+  const tags = {
+    tag100: '100',
+    tag245: '245',
+    tag800: '800',
+    tag660: '660',
+    tag906: '906',
+    tag925: '925',
+    tag955: '955',
+  };
+
+  const inputContent = {
+    field800: '$a Created row',
+    field655primary: '$a Edited 4th field only',
+    field655secondary: '$a Edited MARC tag and 4th field',
+  };
+
+  const expectedSourceText = {
+    row245: '245  1 4  ‡a The Riviera house / ‡c Natasha Lester.',
+    row100: '100  1    ‡a Lester, Natasha, ‡d 1973- ‡e author.',
+    row660: '660    7  ‡a Historical fiction. ‡2 gsafd',
+    row655primary: '655    7  ‡a Edited 4th field only',
+    row655secondary: '655    7  ‡a Edited MARC tag and 4th field',
+    row800: '800       ‡a Created row',
+  };
+
+  const marcFile = {
+    marc: 'marcBibC367956.mrc',
+    fileName: `testMarcFile_C367956_${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+  };
+
+  let instanceId;
+  let testUser;
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
```

Comment:
According to the ticket should be uiQuickMarcQuickMarcBibliographicEditorAll

Reply:
Good catch